lang/cs/Org.Apache.REEF.Utilities/AsyncUtils/LoggingHelper.cs

```diff
@@ -36,8 +37,10 @@ public static void LogAndIgnoreExceptionIfAny(this Task self, Logger logger, str
         {
             self.ContinueWith(t =>
             {
+                var message = string.IsNullOrWhiteSpace(msg) ? string.Empty : msg + " ";
```

Comment:
@markusweimer because if the `string` is empty, we don't want to start with a space - i.e. " Exception...".